### ISSUE TITLE
Bed file validation not following bed specifications

### ISSUE THREAD

**Issue by duartemolha** [brentp/mosdepth#81]
I tried using mosdepth with a bed file that has the fields separated by spaces instead of tabs and it complains:
[mosdepth] skipping bad bed line:chrY 7711640 7711641 rs879195003

This is not correct since a bed tile is white space delimited so both spaces and tabs are acceptable

also, the output of the regions.bed.gz removes part of the input bed file used in --by

for example if my input bed file defining the regions is

chrX    77504877        77508629        ATRX    1       +

the output for the regions bed file is 

chrX    77504877        77508629        ATRX    324.95

The score field and the strand fields are removed. It would be good if they could be retained and the counts just added as an additional column.

Thanks

**Comment by brentp** [comment]
mosdepth requires tab-delimited files. I'd accept a PR to inject the depth into the 5th column when a file has more than 5 columns.

**Comment by duartemolha** [comment]
yes... that would be better to just removing the columns :) thanks
